core/src/test/resources/Http/Responses/simpleMockedHttpGetWithUrlParamAndPath.txt

```diff
@@ -1,5 +1,5 @@
 HOST 127.0.0.1
-GET /value%20value?path=tralala&illegalCharacters=o%40t%26h%3De%2Br%24V%2Ca%2Flue&normalCharacters=helloWorld HTTP/1.1
```

Comment:
Test updated to also test the encoding of special characters in the query param that's part of the configured URL.